### ISSUE TITLE
ETOOMANY issues

### ISSUE THREAD

**Issue by kuboosoft** [megous/megatools#409]
**Describe the bug**
I already reduced the size to 16gb; but megatools fails.. Other I have acces without problem in the web of MEGA and upload without problem.

**To Reproduce**
```
[makerpm@localhost ~]$ megatools ls /Root/urpms/updates/27 --debug=api --reload
-> [{
	"a": "us",
	"user": "xxxxx@gmail.com",
	"uh": "rVyKOkFRoYo"
}]
<- [
	-6
]
ERROR: Can't login to mega.nz: API call 'us' failed: Server returned error ETOOMANY
```
**Expected behavior**
Acces to my account...

**Environment:**
 - Linux Fedora 28
 - **git version also  the 1.10.2**:

**Comment by megous** [comment]
It's probably not file size related (it fails during login). I've never seen this error. (and the description in the API docs looks non-sensical in the context of login)

Are you using a single account from multiple IP addresses at once?

**Comment by kuboosoft** [comment]
Yes, the CI services includes a different IP, maybe a task was uploading files and other download... or uploading the same time. But generally my tasks worked fine... I contacted to mega and also ask me to check my "history"; but yes I have differents IPs I never had the problem...  Ironic, I have full access to my account in the browser... In theory, IPs should not affect; because it is a synchronization service; Today I am in my country I leave my PC turned on, I travel to another country and use my account ... my phone other IP... :sob:

**Comment by kuboosoft** [comment]
finally I deleted my account; I am not sure if is a action to old account with 50GB real; I made a new account and now I have 15GB and some bonus (180 days) enabling some services or sharing the service... Then; megatools works normally again.

The answer from MEGA was to use its client and api... But Its don't work for me...

**Comment by megous** [comment]
Interesting, perhaps they're changing the limits on the new accounts, and you hit some kind of an edge case. I can still see 50GB on my account with 30GB filled. Who knows.

**Comment by kuboosoft** [comment]
I doubt they can maintain accounts with 50GB; for example: Onedrive offers 50GB without limits for $2 per month; of course their free space is 5GB, but it works in a similar way ... For now I am satisfied with the fixed 15GB that MEGA gives, but I need 25Gb aprox.

**Comment by megous** [comment]
I'll probably put the name of this issue on a shirt one day.

**Comment by kuboosoft** [comment]
@megous sure; something so good, it can not be free forever. Maybe I exceeded some limit, hidden for the public ... hahaha

**Comment by kuboosoft** [comment]
Well; If anybody here has the same issue; the problem is because exist too many concurrent IP addresses are accessing your upload target URL... 

This is silly; Because if you use the official clients (MEGAsync, web ...) on a PC that you have in Switzerland for example and you are living in Brazil or use an IC service using "docker" you will lose access to your account; and there is no way to access even with your backup password. Something that does not happen in Dropbox or another service. Security measures? or ignorance?

**Comment by kuboosoft** [comment]
Well; If anybody here has the same issue; the problem is because exist too many concurrent IP addresses are accessing your upload target URL...

This is silly; Because if you use the official clients (MEGAsync, web ...) in a PC that you have in Switzerland for example and you are living in Brazil (using the web) or use an CI service using "docker" you will lose access to your account; and there is no way to access even with your backup password. Something that does not happen in Dropbox or another service. Security measures? or ignorance?

**Comment by megous** [comment]
Unfortunate. Connecting from CI machine can be especially unfortunate, especially if using a thirdparty service that uses many machines with different IPs.

As with any generous free service, you get excessive use, and then the owner eventually starts limiting access. So here we are. From 50G with no quotas down to 15G, 5G quotas, and various incetive programs, and account/IP blocking, etc.

**Comment by kuboosoft** [comment]
I use 10MB-1GB approx max (sporadic) in my account (isn't excessive), reduced to 10MB because my files (rpms linux) are temporal (I use it as a backup temporally). Then my traffic is low; detecting multiple IPs is bad for all but not logic, don't matter if its is a free account. Imagine if I pay a PRO account, It will be the same results and is easy lost your files. Megatools is a good client; I don't like the official cli, because is impossible the automations.

**Comment by megous** [comment]
I was speaking in general, not about you. People who overuse the free service spoil the party for everyone in the end. It's hard to determine who uses the service nicely and who just got his 50th IP address to avoid quotas. You almost always have to use some heuristics, and nice users will get blocked too, if they stray further from the norm.

I guess mega's not too excited either, to block nice free users, because it lessens trust in their service.
